### ISSUE TITLE
UselessParentheses false positive in divisions

### ISSUE THREAD

**Issue by mauriciofauth** [slevomat/coding-standard#800]
```php
$a = 10 / (5 / 2); // double(4)

$a = 10 / 5 / 2; // int(1)
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/bb8be4374e657b131c5ea0de52105408799e998c
